### DISCUSSION TITLE
Random posts with 'follow' option tag on mobile facebook

### DISCUSSION THREAD

**Discussion by macerg** [uBlockOrigin/uAssets#23411] [Q&A] [answered]
Hello, not raising as an 'issue' yet as may already ha a 'fix' posted. I am on Firefox m.facebook.com.
Getting random posts like this. Is there a fix please?
![Uploading Screenshot_20240423-082847.jpg…]()

**Comment by macerg** [comment]
![Screenshot_20240423-082847](https://github.com/uBlockOrigin/uAssets/assets/4559048/edd954ba-8373-41f7-abe8-449b86608bd8)

**Reply by u-RraaLL** [reply, marked as answer]
Facebook treats `Home` the same way twitter treats `For you`. It's a timeline with suggestions.

To view posts only from people/things you actually follow you need to open `Feeds`.

Unfortunately, mobile FB is coded terribly and doesn't allow a direct link to reach it. You have to open the hamburger menu and choose Feeds every time.

These posts can't really be filtered on mobile, because there's too many of them and their removal will break scrolling after a couple of posts.
